tests/integration/testlib/ovslib.py

```diff
@@ -58,9 +68,11 @@ def add_internal_port(self, name, ipv4_state):
         )
 
     def _add_port(self, name):
+        port = {OVSBridge.PORT_NAME: name}
         self._bridge_iface[OVSBridge.CONFIG_SUBTREE].setdefault(
             OVSBridge.PORT_SUBTREE, []
-        ).append({OVSBridge.PORT_NAME: name})
+        ).append(port)
+        return port
```

Comment:
it is not recommended to do both setting and getting in the same method.
It may be convenient when used, but it causes other problems.
The method changes the state. The caller can fetch the new state or re-calculate `{OVSBridge.PORT_NAME: name}`.